src/fairseq2/utils/__init__.py

```diff
@@ -0,0 +1,3 @@
+from fairseq2.utils.read_and_pickle_wrap_iterator import (
```

Comment:
I would prefer this helper to reside under `fairseq2.data.utils` instead of `fairseq2.utils`.

Reply:
See the latest change - let me know how exactly you want the path to look like.